criu/img-remote.c

```diff
@@ -940,11 +940,20 @@ static int64_t send_image_async(struct roperation *op)
 	struct rimage *rimg = op->rimg;
 	bool close_fd = op->close_fd;
 	int n;
+	int flags = 0;
+
+#ifdef MSG_ZEROCOPY
+	flags |= MSG_ZEROCOPY;
+#endif
 
-	n = write(
+	n = send(
 		fd,
 		op->curr_sent_buf->buffer + op->curr_sent_bytes,
-		min(BUF_SIZE, op->curr_sent_buf->nbytes) - op->curr_sent_bytes);
```

Comment:
Now op->curr_sent_buf->buffer can't be modifies immediately after this call. We need a comment here which explains life-time of this buffer.